src/gui/src/dbDescriptors.cpp

```diff
@@ -1639,4 +1648,139 @@ bool DbTechLayerDescriptor::getAllObjects(SelectionSet& objects) const
   return true;
 }
 
+//////////////////////////////////////////////////
+
+static const char*
+getAccessTypeName(odb::dbAccessPoint::AccessType type)
+{
+  switch(type) {
+  case odb::dbAccessPoint::OnGrid:
+    return "OnGrid";
+  case odb::dbAccessPoint::HalfGrid:
+    return "HalfGrid";
+  case odb::dbAccessPoint::Center:
+    return "Center";
+  case odb::dbAccessPoint::EncOpt:
+    return "EncOpt";
+  case odb::dbAccessPoint::NearbyGrid:
+    return "NearbyGrid";
+  }
+  return "Unknown";
+}
+
+DbItermAccessPointDescriptor::DbItermAccessPointDescriptor(odb::dbDatabase* db) :
+    db_(db)
+{
+}
+
+std::string DbItermAccessPointDescriptor::getName(std::any object) const
+{
+  auto iterm_ap = std::any_cast<DbItermAccessPoint>(object);
+  auto ap = iterm_ap.ap;
+  std::string name(getAccessTypeName(ap->getLowType()));
+  name += std::string("/") + getAccessTypeName(ap->getHighType());
+  return name;
+}
+
+std::string DbItermAccessPointDescriptor::getTypeName() const
+{
+  return "Access Point";
+}
+
+bool DbItermAccessPointDescriptor::getBBox(std::any object, odb::Rect& bbox) const
+{
+  auto iterm_ap = std::any_cast<DbItermAccessPoint>(object);
+  odb::Point pt = iterm_ap.ap->getPoint();
+  odb::dbTransform xform;
+  int x, y;
+  iterm_ap.iterm->getInst()->getLocation(x, y);
+  xform.setOffset({x, y});
+  xform.apply(pt);
+  bbox = {pt, pt};
+  return true;
+}
+
+void DbItermAccessPointDescriptor::highlight(std::any object,
+                                        Painter& painter,
+                                        void* additional_data) const
+{
+  auto iterm_ap = std::any_cast<DbItermAccessPoint>(object);
+  odb::Point pt = iterm_ap.ap->getPoint();
+  odb::dbTransform xform;
+  int x, y;
+  iterm_ap.iterm->getInst()->getLocation(x, y);
+  xform.setOffset({x, y});
+  xform.apply(pt);
+  const int shape_size = 100;
+  painter.drawLine({pt.x() - shape_size / 2, pt.y() - shape_size / 2},
+                   {pt.x() + shape_size / 2, pt.y() + shape_size / 2});
+  painter.drawLine({pt.x() - shape_size / 2, pt.y() + shape_size / 2},
+                   {pt.x() + shape_size / 2, pt.y() - shape_size / 2});
+}
+
+Descriptor::Properties DbItermAccessPointDescriptor::getProperties(std::any object) const
+{
+  auto iterm_ap = std::any_cast<DbItermAccessPoint>(object);
+  auto ap = iterm_ap.ap;
+
+  std::vector<odb::dbDirection> accesses;
+  ap->getAccesses(accesses);
+
+  std::string directions;
+  for (auto dir : accesses) {
+    if (!directions.empty()) {
+      directions += ", ";
+    }
+    directions += dir.getString();
+  }
+
+  Properties props({{"low type", getAccessTypeName(ap->getLowType())},
+                    {"high type", getAccessTypeName(ap->getHighType())},
+                    {"directions", directions},
+                    {"layer", ap->getLayer()->getName()}
```

Comment:
The other DBDescriptors use "Low Type" "High Type" "Directions", "Layer". I would keep that constant